.github/workflows/ci-core.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           gc-basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
           gc-host: ${{ secrets.GRAFANA_CLOUD_HOST }}
       - name: Notify Slack
-        if: ${{ failure() && github.event.schedule != '' }}
+        if: ${{ failure() && (github.event_name == 'merge_group' || github.event.branch == 'develop')}}
```

Comment:
Would it be easier to invert this and to instead ignore any pull request events?

Reply:
I was worried about including release and master since we sometimes have weird situations there.